pact4s/src/main/scala/org/broadinstitute/dsde/workbench/cromwell/consumer/CbasClient.scala

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.dsde.workbench.cromwell.consumer
+
+import cats.effect.Concurrent
+import cats.syntax.all._
+import com.typesafe.scalalogging.LazyLogging
+import io.circe.Encoder
+import org.http4s._
+import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+import org.http4s.client.Client
+import org.http4s.headers.Authorization
+
+final case class WorkflowCallbackMessage(workflowId: String,
```

Comment:
I might be misunderstanding exactly what this is supposed to be testing, but - should we be using logic and data structures from the actual workflow callback code here, rather than creating a parallel implementation? I'm not sure what this is doing to test Cromwell's half of the contract.

Reply:
I created a CbasClient template to validate the cromwell-cbas contract testing on pact broker. It works fine - for this iteration.

Reply:
I'm tempted to agree with Janet, could we reuse the same message generation code that Cromwell uses internally to build this message, rather than re-building it separately here? Eg if Cromwell was building the requests in a way that CBAS didn't like, we wouldn't find that out if _this_ message builder was doing things correctly.